classes/classes/Items/MutationsHelper.as

```diff
@@ -190,6 +188,43 @@ package classes.Items
 			return "invalid"; // Will never happen. Suppresses 'Error: Function does not return a value.'
 		}
 
+		public function updateClaws(clawType:int = CLAW_TYPE_NORMAL):String
+		{
+			var clawTone:String = "";
+			var oldClawTone:String = player.clawTone;
+
+			switch (clawType) {
+				case CLAW_TYPE_DRAGON:       clawTone = "steel-gray";   break;
+				case CLAW_TYPE_SALAMANDER:   clawTone = "fiery-red";    break;
+				case CLAW_TYPE_LIZARD:
+					// See http://www.bergenbattingcenter.com/lizard-skins-bat-grip/ for all those NYI! lizard skin colors
+					// I'm still not that happy with these claw tones. Any suggestion would be nice.
+					switch (player.skinTone) {
+						case "red":          clawTone = "reddish";      break;
+						case "green":        clawTone = "greenish";     break;
+						case "white":        clawTone = "light-gray";   break;
+						case "blue":         clawTone = "bluish";       break;
+						case "black":        clawTone = "dark-gray";    break;
+						case "purple":       clawTone = "purplish";     break;
+						case "silver":       clawTone = "silvery";      break;
+						case "pink":         clawTone = "pink";         break; // NYI! Maybe only with a new Skin Oil?
+						case "orange":       clawTone = "orangey";      break; // NYI!
```

Comment:
I just found out, that you can gain orangey claws, when you TF towards being an imp. Imps don't change their skinType and can have orange skin/fur/scales.